### PR TITLE
glibc update to 2.26

### PR DESCRIPTION
**Pull request by lrusak**
this is for after #1664 as the gcc patches are for gcc 7.1.0

### CODE REVIEW

File: packages/devel/glibc/package.mk [hunk 1]
```diff
@@ -50,6 +50,7 @@ PKG_CONFIGURE_OPTS_TARGET="BASH_SHELL=/bin/sh \
                            --without-cvs \
                            --without-gd \
                            --enable-obsolete-rpc \
+                           --enable-obsolete-nsl \
```
Comment (stefansaraev) [inline packages/devel/glibc/package.mk]: I dont think you need this.

Comment (lrusak) [inline packages/devel/glibc/package.mk]: I do otherwise rpcbind bails out

**Comment by InuSasha** [comment]
@lrusak are the gcc patches from upstream? (does not apply with gcc 7.2)

**Comment by lrusak** [comment]
Included in #1164

**Pull request closed without merging**